.github/workflows/continuous-integration.yaml

```diff
@@ -510,13 +510,15 @@ jobs:
               - tests/seeding/test_seeding.py
             docs:
               - docs/**
+              - '!docs/developer-guide/api/**.md'
```

Comment:
not sure about this one honestly , that would skip `make docs` in case only apidocs were updated (and I don't think that's a good idea)